hardinfo/info.c

```diff
@@ -40,7 +40,7 @@ void info_group_add_fieldsv(struct InfoGroup *group, va_list ap)
     while (1) {
         struct InfoField field = va_arg(ap, struct InfoField);
 
-        if (!field.name)
```

Comment:
Any reason to add a magic field and not rely on field.name being NULL?

Reply:
I was just looking for a use for that .some_member_with_default_value, it was just an idea.

Reply:
I reverted that change.